examples/android/client/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Flower</string>
+    <string name="notification_channel_id">1002</string>
+    <string name="notification_title">FL Background Worker On</string>
+    <string name="cancel_download">cancel the FL process </string>
+    <string name="channelId">123</string>
```

Comment:
Why 123? Whats the significance of this?

Reply:
This has been an unnecessary variable. I will remove it. Sorry for the inconvenience.